Makefile

```diff
@@ -142,14 +142,13 @@ clean:
 	$(GO) clean -tags '$(TAGS)' $(GOFLAGS) -i github.com/cockroachdb/...
 	find . -name '*.test' -type f -exec rm -f {} \;
 	rm -rf build/deploy/build
-# List all of the dependencies which are not part of the standard
-# library or cockroachdb/cockroach.
-.PHONY: listdeps
-listdeps:
```

Comment:
so listdeps doesn't actually list the commands either - those are installed by `glock sync`